src/pages/saksbehandling/revurdering/formue/RevurderFormueUtils.ts

```diff
@@ -0,0 +1,264 @@
+import * as DateFns from 'date-fns';
+
+import * as DateUtils from '~lib/dateUtils';
+import { Nullable } from '~lib/types';
+import yup, { validateStringAsNonNegativeNumber } from '~lib/validering';
+import { Formuegrenser, FormueVilkår } from '~types/grunnlagsdataOgVilkårsvurderinger/formue/Formuevilkår';
+import { FormuegrunnlagFormue, FormuegrunnlagVerdier } from '~types/Revurdering';
+
+export interface FormueFormData {
+    formue: FormueData[];
+}
+export interface FormueData {
+    epsFnr: Nullable<string>;
+    periode: { fraOgMed: Nullable<Date>; tilOgMed: Nullable<Date> };
+    søkersFormue: VerdierFormData;
+    epsFormue: Nullable<VerdierFormData>;
+    begrunnelse: Nullable<string>;
+}
+export interface VerdierFormData {
+    verdiPåBolig: string;
+    verdiPåEiendom: string;
+    verdiPåKjøretøy: string;
+    innskuddsbeløp: string;
+    verdipapir: string;
+    kontanterOver1000: string;
+    stårNoenIGjeldTilDeg: string;
+    depositumskonto: string;
+}
+
+const tomFormue: VerdierFormData = {
+    verdiPåBolig: '0',
+    verdiPåEiendom: '0',
+    verdiPåKjøretøy: '0',
+    innskuddsbeløp: '0',
+    verdipapir: '0',
+    kontanterOver1000: '0',
+    stårNoenIGjeldTilDeg: '0',
+    depositumskonto: '0',
+};
+
+export const leggTilNyPeriode = (epsFnr: Nullable<string>): FormueData => {
```

Comment:
Litt merkelig navn på denne. Hadde det gitt mening med f.eks. 
```suggestion
export const getTomFormueData = (epsFnr: Nullable<string>): FormueData => {
```
eller noe sånt?